src/services/common/measurement.utils.ts

```diff
@@ -91,9 +91,14 @@ const getFormattedPoint = function (point: Point, decimals: number) {
     .map(c => c.toPrecision(decimals))
     .join(' ')
 }
+
 export {
-  getFormattedLength,
-  getFormattedArea,
+  getLength,
+  getArea,
```

Comment:
Maybe we should have one getArea and one getLength (overloaded) functions that can take any geom as a param and do the dispatch according to the type in this utils (instead of in the composable or component, but it is working so you can keep it as is if you think that's too much time for a rework or if it is not necessary)